rct229/rules/section23/section23rule3.py

```diff
@@ -0,0 +1,83 @@
+
+APPLICABLE_SYS_TYPES = [
+    HVAC_SYS.SYS_5,
+    HVAC_SYS.SYS_6,
+    HVAC_SYS.SYS_7,
+    HVAC_SYS.SYS_8,
+    HVAC_SYS.SYS_11_1,
+]
+
+
+class Section23Rule3(RuleDefinitionListIndexedBase):
+    """Rule 3 of ASHRAE 90.1-2019 Appendix G Section 23 (Air-side)"""
+
+    def __init__(self):
+        super(Section23Rule3, self).__init__(
+            rmrs_used=UserBaselineProposedVals(False, True, False),
+            each_rule=Section23Rule3.TerminalRule(),
+            index_rmr="baseline",
+            id="23-3",
+            description="System 5, 6, 7 and 8 minimum volume setpoint shall be 30% of zone peak airflow, minimum outdoor airflow, or rate required to comply with minium accreditation standards whichever is larger.",
+            ruleset_section_title="HVAC - Airside",
+            standard_section="Section G3.1.3.13 VAV Minimum Flow Set Points (Systems 5 and 7) and Section G3.1.3.14 Fan Power and Control (Systems 6 and 8)",
+            is_primary_rule=True,
+            rmr_context="ruleset_model_instances/0",
+            list_path="$..terminals[*]",
+        )
+
+    def is_applicable(self, context, data=None):
+        rmi_b = context.baseline
+        baseline_system_types_dict = get_baseline_system_types(rmi_b)
+
+        return any(
+            [
+                baseline_system_type_compare(system_type, applicable_sys_type, False)
+                for system_type in baseline_system_types_dict.keys()
+                for applicable_sys_type in APPLICABLE_SYS_TYPES
+            ]
+        )
+
+    class TerminalRule(RuleDefinitionBase):
+        def __init__(self):
+            super(Section23Rule3.TerminalRule, self).__init__(
+                rmrs_used=UserBaselineProposedVals(False, True, False),
+                required_fields={
+                    "$": [
+                        "minimum_airflow",
+                        "minimum_outdoor_airflow",
+                        "primary_airflow",
+                    ],
+                },
+            )
+
+        def get_calc_vals(self, context, data=None):
+            terminal_b = context.baseline
+            minimum_airflow_b = terminal_b["minimum_airflow"]
+            primary_airflow_b = terminal_b["primary_airflow"]
+            minimum_outdoor_airflow_b = terminal_b["minimum_outdoor_airflow"]
+
+            return {
+                "minimum_airflow_b": CalcQ("volumetric_flow_rate", minimum_airflow_b),
+                "primary_airflow_b": CalcQ("volumetric_flow_rate", primary_airflow_b),
+                "minimum_outdoor_airflow_b": CalcQ(
+                    "volumetric_flow_rate", minimum_outdoor_airflow_b
+                ),
+            }
+
+        def rule_check(self, context, calc_vals=None, data=None):
+            minimum_airflow_b = calc_vals["minimum_airflow_b"]
+            primary_airflow_b = calc_vals["primary_airflow_b"]
+            minimum_outdoor_airflow_b = calc_vals["minimum_outdoor_airflow_b"]
+
+            return minimum_airflow_b == max(
```

Comment:
I think we'll need to use `std_equal()` or something like it here because multiplying by `0.3` is not exact.

Reply:
Great point! Fixed.